src/cljs/athens/events.cljs

```diff
@@ -609,14 +609,27 @@
 
 
 (reg-event-fx
-  :theme/toggle
+  :theme/set-dark
   (fn [{:keys [db]} _]
-    (let [dark?    (:theme/dark db)
-          new-dark (not dark?)
-          theme    (if dark? style/THEME-LIGHT style/THEME-DARK)]
-      {:db                 (assoc db :theme/dark new-dark)
-       :local-storage/set! ["theme/dark" new-dark]
-       :stylefy/tag        [":root" (style/permute-color-opacities theme)]})))
+    {:db                 (assoc db :theme/dark true)
+     :local-storage/set! ["theme/dark" true]
+     :stylefy/tag        [":root" (style/permute-color-opacities style/THEME-DARK)]}))
+
+
+(reg-event-fx
+  :theme/set-light
+  (fn [{:keys [db]} _]
+    {:db                 (assoc db :theme/dark false)
+     :local-storage/set! ["theme/dark" false]
+     :stylefy/tag        [":root" (style/permute-color-opacities style/THEME-LIGHT)]}))
```

Comment:
@shanberg

Reply:
Actually `:theme/toggle` is not split into 2 but it was removed from the codebase and `set-dark` and `set-light` took its place. Because `:theme/toogle` is not used in the codebase now.